tip-wlan/charts/opensync-gw-cloud/templates/deployment.yaml

```diff
@@ -34,6 +34,17 @@ spec:
           - "-service={{ .Release.Name }}-wlan-prov-service"
           - "-service={{ .Release.Name }}-wlan-ssc-service"
           - -check_interval=5
+        - name: download-jmx-prometheus-exporter
+          image: alpine:latest
+          command:
+          - wget
+          args:
+          - -P
+          - /jmx-prometheus-exporter-dir
+          - https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/0.14.0/jmx_prometheus_javaagent-0.14.0.jar
```

Comment:
Better to point this to a mirror in the JFrog, instead of the maven central. I know, it's a prototype, but there's nothing more permanent than a temporary thing :)